subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -367,7 +367,7 @@ static int common_prepare_cb(struct lll_prepare_param *p, bool is_resume)
 
 	aa = sys_cpu_to_le32(PDU_AC_ACCESS_ADDR);
 	radio_aa_set((uint8_t *)&aa);
-	radio_crc_configure(((0x5bUL) | ((0x06UL) << 8) | ((0x00UL) << 16)),
+	radio_crc_configure(RADIO_CRC_POLYNOMIAL,
 			    0x555555);
```

Comment:
`0x555555` could also be defined in `pdu.h`

Reply:
I will define it as PDU_AC_CRC_IV in pdu.h as well